src/api.ts

```diff
@@ -334,3 +345,26 @@ export class MirrordAPI {
         });
     }
 }
+
+
+/** 
+* Updates the global feedback counter. When it hits `FEDBACK_COUNTER_REVIEW_AFTER` mirrord runs, 
+* displays a message asking the user to like mirrord.
+*/ 
+function tickFeedbackCounter() {
+    const counter = parseInt(globalContext.globalState.get(FEEDBACK_COUNTER) ?? '0') || 0;
+    globalContext.globalState.update(FEEDBACK_COUNTER, `${counter + 1}`);
```

Comment:
Is the stringification necessary? Doesn't `update` stringify for you?

Reply:
Good question, I just copied this from the other instance where we change numbers in `globalState`, will check why it was done this way.